_episodes/12-cluster.md

```diff
@@ -139,6 +139,16 @@ node*, *login node*, *landing pad*, or *submit node*. A login node serves as an
 cluster. As a gateway, it is well suited for uploading and downloading files, setting up software,
 and running quick tests. It should never be used for doing actual work.
 
+> ## Dedicated Transfer Nodes
+>
+> If you want to transfer larger amounts of data to or from the cluster, some systems offer
+> dedicated nodes for data transfers only. The motivation for this lies in the fact, that larger
+> data transfers should not obstruct operation of the login node for anybody else. Check with your
+> cluster's documentation or its support team, if such a transfer node is available. As a rule of
+> thumb, consider all transfers of a volume larger than 500MB to 1GB as large. But these numbers
```

Comment:
- While mistaken, the "rule of thumb" is believed by some to have an [offensive etymology](https://en.wikipedia.org/wiki/Rule_of_thumb).
- For any modern network/filesystem combination, I would consider anything that fits on a reasonably accessible USB stick to be a small data transfer. If you're moving more than 100 GB, then the data transfer node is probably better suited.
- A different perspective, perhaps more meaningful, is the *bandwidth* impact on other users. You can transfer anything you want through the landing pad, so long as you throttle it to ~2 MB/s. If you want an *unrestricted* transfer, which may be necessary if you hope for it to finish in your lifetime, use the data transfer node.
- All of this depends on the hardware and network you're transferring over. If you're moving something over InfiniBand, just go for it.